### PR TITLE
Release 2.10.6

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**: Release

**Context**:  Release 2.10.6

**Summary**: As usual :rocket:

**Acceptance criteria**:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # CHANGELOG
-## Release 2.10.6 (in development)
+## Release 2.10.7(in development)
```
Comment (gdemonet) [inline CHANGELOG.md]: nit: we're missing a space! :smile: 

```suggestion
## Release 2.10.7 (in development)
```

Comment (ChengYanJin) [reply CHANGELOG.md]: good catch !!!

**Review by gdemonet — APPROVED** [review]

**Pull request merged**